react/queries/product.gql

```diff
@@ -82,7 +82,6 @@ query Product(
           }
           Price
           ListPrice
-          spotPrice
```

Comment:
ops, i dont think this should be reverted

Reply:
ahh, good catch

Reply:
(was still reviewing my own pr tho hahah)